README.md

```diff
@@ -1,219 +1,268 @@
+```
 
-```javascript
-import appInsights = require("applicationinsights");
+More info on the telemetry API is available in [the docs][].
 
-// configure auto-collection with one instrumentation key
-appInsights.setup("<instrumentation_key>").start();
+[the docs]: https://azure.microsoft.com/documentation/articles/app-insights-api-custom-events-metrics/
 
-// get a client for another instrumentation key
-var otherClient = appInsights.getClient("<other_instrumentation_key>");
-otherClient.trackEvent("custom event");
-```
 
-## Examples
+## Use multiple instrumentation keys
 
-### Tracking dependency
+You can create multiple Azure Application Insights resources and send different
+data to each by using their respective instrumentation keys ("ikey"). For
+example:
 
 ```javascript
-import appInsights = require("applicationinsights");
-var client = appInsights.getClient();
+let appInsights = require("applicationinsights");
 
-var startTime = Date.now();
-// execute dependency call
-var endTime = Date.now();
+// configure auto-collection under one ikey
+appInsights.setup("_ikey-A_").start();
 
-var elapsedTime = endTime - startTime;
-var success = true;
-client.trackDependency("dependency name", "command name", elapsedTime, success);
+// track some events manually under another ikey
+let otherClient = appInsights.getClient("_ikey-B_");
+otherClient.trackEvent("my custom event");
 ```
 
+## Examples
 
+* Track dependencies
 
-### Manual request tracking of all "GET" requests
+    ```javascript
+    let appInsights = require("applicationinsights");
+    let client = appInsights.getClient();
 
-```javascript
-var http = require("http");
-var appInsights = require("applicationinsights");
-appInsights.setup("<instrumentation_key>")
-    .setAutoCollectRequests(false) // disable auto-collection of requests for this example
-    .start();
+    var success = false;
+    let startTime = Date.now();
+    // execute dependency call here....
+    let duration = Date.now() - startTime;
+    success = true;
 
-// assign common properties to all telemetry sent from the default client
-appInsights.client.commonProperties = {
-    environment: process.env.SOME_ENV_VARIABLE
-};
+    client.trackDependency("dependency name", "command name", duration, success);
+    ```
 
-// track a system startup event
-appInsights.client.trackEvent("server start");
+* Assign custom properties to be included with all events
 
-// create server
-var port = process.env.port || 1337
-var server = http.createServer(function (req, res) {
-    // track all "GET" requests
-    if(req.method === "GET") {
-        appInsights.client.trackRequest(req, res);
-    }
+    ```javascript
+    appInsights.client.commonProperties = {
+      environment: process.env.SOME_ENV_VARIABLE
+    };
+    ```
 
-    res.writeHead(200, { "Content-Type": "text/plain" });
-    res.end("Hello World\n");
-}).listen(port);
+* Manually track all HTTP GET requests

```

Comment:
Probably mention that in this case you want to set `` .setAutoCollectRequests(true)``?